extractor/extractor.go

```diff
@@ -0,0 +1,266 @@
+package extractor
+
+import (
+	"bufio"
+	"fmt"
+	"go/ast"
+	"go/parser"
+	"go/token"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"github.com/vision-cli/common/cases"
+	"github.com/vision-cli/common/transpiler/model"
+	protoparser "github.com/yoheimuta/go-protoparser"
+	parserStructs "github.com/yoheimuta/go-protoparser/parser"
+)
+
+func ProjectStructure(projectDirectory string) ([]model.Module, error) {
+	targetDir := filepath.Join(projectDirectory, "services")
```

Comment:
pls don't hard code "services", it should be a parameter passed in (in other cases it could be a constant, but in this case the services directory is variable so should be passed in)